tools/release/verify_python_release.py

```diff
@@ -71,9 +71,11 @@ def _get_remote_artifacts_for_package(package, version):
     experience, it has taken a minute on average to be fresh.
     """
     artifacts = set()
-    payload = requests.get("https://pypi.org/pypi/{}/{}/json".format(
-        package, version)).json()
-    for download_info in payload['releases'][version]:
+    payload_resp = requests.get("https://pypi.org/pypi/{}/{}/json".format(
```

Comment:
What does resp means?

Reply:
It's a pretty common abbreviation for `response`. Similarly, `req` is often used instead of `request`